target/linux/armsr/base-files/lib/upgrade/platform.sh

```diff
@@ -34,10 +34,10 @@ platform_check_image() {
 }
 
 platform_copy_config() {
-	local partdev parttype=ext4
+	local partdev
 
-	if export_partdevice partdev 2; then
-		mount -t $parttype -o rw,noatime "/dev/$partdev" /mnt
+	if export_partdevice partdev 1; then
+		mount -t vfat -o rw,noatime "/dev/$partdev" /mnt
```

Comment:
No... check x86... part_magic_fat should be used instead of hardcoding... Also can you explain why 2 was changed to 1?

Reply:
Partition 1 is the boot partition whereas partition 2 is either ext4 or squashfs. If it's the latter it wouldn't be writable.